### PR TITLE
Add SavingAsXlsx

### PR DESCRIPTION
**Pull request by RePreus**
<3

### CODE REVIEW

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Office.Interop.Excel;
+using Portierniaktosiedzi.Models;
```
Comment (Kenstin) [inline Portierniaktosiedzi/Models/SavingAsXlsx.cs]: unnecessary usings

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace Portierniaktosiedzi.Extensions
```
Comment (Kenstin) [inline Portierniaktosiedzi/Models/SavingAsXlsx.cs]: wrong namespace

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+        public SavingAsXlsx(string path, /*NegativeArray<Day> list,*/ string name)
+        {
+            timetable = new Application();
+            workbook = timetable.Workbooks.Add(XlWBATemplate.xlWBATWorksheet);
+            worksheet = (Worksheet)workbook.Worksheets[0];
+            worksheet.Name = "Harmonogram";
+            GenerateTemplateSheet();
+            object missing = System.Reflection.Missing.Value;
+            workbook.SaveAs(
```
Comment (Kenstin) [inline Portierniaktosiedzi/Models/SavingAsXlsx.cs]: workbook.SaveAs(Filename: "kupa", );

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace Portierniaktosiedzi.Extensions
+{
+    class SavingAsXlsx
+    {
+        private static Application timetable;
+        private Workbook workbook;
+        private Worksheet worksheet;
+
+        public SavingAsXlsx(string path, /*NegativeArray<Day> list,*/ string name)
```
Comment (Kenstin) [inline Portierniaktosiedzi/Models/SavingAsXlsx.cs]: SaveAsXlsx

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        private void GenerateTemplateSheet()
+        {
+            AdjustColumnWidth();
+            MergeRows();
+        }
+
+        private void AdjustColumnWidth()
+        {
+            int[] tab = new int[] { 14, 13, 17, 17, 17, 17, 17, 17, 17, 17, 17 };
+            for (int i = 0; i < 11; i++)
```
Comment (Kenstin) [inline Portierniaktosiedzi/Models/SavingAsXlsx.cs]: `tab.Length`

**Review by Kenstin — CHANGES_REQUESTED** [review]

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs (hunk 4 above)
Comment (Kenstin) [reply Portierniaktosiedzi/Models/SavingAsXlsx.cs]: XlsxGenerator?

File: Portierniaktosiedzi/Models/SavingAsXlsx.cs (hunk 1 above)
Comment (jodua) [reply Portierniaktosiedzi/Models/SavingAsXlsx.cs]: kenstin jest super

Comment (RePreus) [reply Portierniaktosiedzi/Models/SavingAsXlsx.cs]: you are unnecessary lol

**Review by Kenstin — APPROVED** [review]

**Pull request merged**